install/update_946_947.php

```diff
@@ -46,6 +46,14 @@ function update946to947() {
 
    $DB->updateOrDie('glpi_events', ['type'   => 'dcrooms'], ['type' => 'serverroms']);
 
+   $migration->dropField(ReservationItem::getTable(), "is_deleted");
+   $migration->addKey(
```

Comment:
Migration will fail if there is already multiple row for one item.
Deduplication should try to not alter existing behaviour, if possible.